### ISSUE TITLE
Support --with-{gmp,mpfr,mpc} . Again. But on Fedora 33 riscv64, under qemu. Targeting Clang. gollvm

### ISSUE THREAD

**Issue by advancedwebdeveloper** [riscv/riscv-tools#327]
Hi.
I am curious whether there is a support of D and F extensions (as I understood from the publications) in qemu.

I have a project here, on a match https://go.googlesource.com/gollvm/ - and it has these libraries as dependencies.
Currently looking for reasonable alternatives (maybe even getting rid of those, on RISC-V, to rely on arch. specific instructions).

https://github.com/riscv/riscv-tools/issues/72 describes my current situation pretty well.

CC @bschommer , in case if he would be interested.

**Comment by jim-wilson** [comment]
Yes, qemu supports D and F.  You still need gmp, mpfr, and mpc.  But they will be compiled to use D and F instructions.  Since you have Fedora, just use dnf to install the libraries if you don't already have them: gmp-devel, mpfr-devel, mpc-devel.
